components/usage/pkg/db/cost_center.go

```diff
@@ -263,3 +263,50 @@ func (c *CostCenterManager) ListLatestCostCentersWithBillingTimeBefore(ctx conte
 
 	return results, nil
 }
+
+func (c *CostCenterManager) ResetUsage(ctx context.Context, cc CostCenter) (CostCenter, *Usage, error) {
```

Comment:
we are doing most of the logic in `upgradeToStripe`, which is ill-named as it doesn't to anything specific to stripe but resets the usage. Would be great if you merge those functions.